### PR TITLE
Partner segments

### PR DESCRIPTION
**Pull request by cloveras**

### CODE REVIEW

**Review by rebekaburnett — APPROVED** [review]
Awesome! So fast!

**Comment by matsjohans** [comment]
Discussed with Ana and we need to wait a bit communicating this, since we just decided to change names for the different tiers and content per tier may change next days

**Comment by rebekaburnett** [comment]
I found out, after some searching, that "kickbacks" => "partner commissions" :-)

**Review by rebekaburnett — APPROVED** [review]

**Review by rebekaburnett — APPROVED** [review]

**Review by rebekaburnett — APPROVED** [review]

File: add-portal-user.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!-- START_METADATA
 ---
 title: How to add a user on the portal
-sidebar_position: 2
+sidebar_position: 25
```
Comment (cloveras) [inline add-portal-user.md:4]:
```suggestion
sidebar_position: 30
```

**Pull request merged**